### PR TITLE
Naive baseline processor text

### PR DESCRIPTION
**Pull request by EhrenbruderRoschtbraten**
Implemented the naive baseline processor for texts. 

Should be merged after the image rename branch.

### CODE REVIEW

File: src/fool_ai_detector/service/naive_baseline_processor_text.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+from fool_ai_detector.model.processor import Processor
+
+
+class NaiveBaselineProcessorText(Processor):
+    def process(self, input_file_path, output_file_path):
+        """
+        Replaces every occurrence of a white space with double white space to augment a text.
```
Comment (LeviUniAcc) [inline src/fool_ai_detector/service/naive_baseline_processor_text.py:9]:
```suggestion
        Replaces every occurrence of a whitespace with double whitespace to augment a text.
```

**Review by LeviUniAcc — APPROVED** [review]
test works, lgtm

**Review by RaphaW — APPROVED** [review]
LGTM and works for me

**Comment by EhrenbruderRoschtbraten** [comment]
> Can you recreate the PR or push a little commit to update it? The view on GitHub still includes the changes compared to before the renaming PR was merged

Will recreate it.

**Comment by EhrenbruderRoschtbraten** [comment]
> Can you recreate the PR or push a little commit to update it? The view on GitHub still includes the changes compared to before the renaming PR was merged

Reopened it, because it didn't work with creating a new PR. Also didn't work with pushing a commit.

File: test/fool_ai_detector/service/test_naive_baseline_processor_text.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        Helper method: This method creates the test folders.
+        """
+        if not os.path.exists(self.output_processor_directory_path):
+            os.makedirs(self.output_processor_directory_path)
+
+    def test_process(self):
+        self.set_up()
+
+        naive_baseline_processor = NaiveBaselineProcessorText()
+        naive_baseline_processor.process(self.output_generator_file_path, self.output_processor_file_path)
```
Comment (1Maxnet1) [inline test/fool_ai_detector/service/test_naive_baseline_processor_text.py:27]: I need to create the `processed_alpen.txt` file, else the test will fail here. Could we change it so it does create the file, if it does not exist yet?

**Review by 1Maxnet1 — COMMENTED** [review]
One issue, otherwise fine for me

**Pull request merged**